packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/pg-string-datasource-adapter.test.ts.snap

```diff
@@ -279,3 +279,38 @@ Array [
   },
 ]
 `;
+
+exports[`testPostgresStringDataSourceAdapter sets the model field correctly from a schema with a SERIAL field 1`] = `
```

Comment:
This behavior is not new but I am codifying it since generation depends on it now.